### PR TITLE
Added telemetry  for new packages

### PR DESCRIPTION
**Pull request by manangoel99**
Fixes WB-9847

Description
-----------
The PR adds new import telemetry.

Testing
-------
Added functional tests.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -6,7 +6,8 @@ envlist = black,
           docstrings,
           py{36,37,38,39,launch},
           func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,docs,
-                  imports1,imports2,imports3,imports4,imports5,imports6,noml,grpc}-py37,
+                  imports1,imports2,imports3,imports4,imports5,imports6,
+                  imports7,imports8,imports9,imports10,imports11,noml,grpc}-py37,
```
Comment (dmitryduev) [inline tox.ini:10]:
```suggestion
                  imports{1,2,3,4,5,6,7,8,9,10,11},noml,grpc}-py37,
```

File: tox.ini [hunk 2]
```diff
@@ -6,7 +6,8 @@ envlist = black,
           docstrings,
           py{36,37,38,39,launch},
           func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,docs,
```
Comment (dmitryduev) [inline tox.ini:8]: while you're at it, mind also doing `tf{115,21,25,26}` please?

File: tox.ini (hunk 1 above)
Comment (dmitryduev) [reply tox.ini:10]: I think tox supports nested substitutions, let's give it a spin and see :)

Comment (dmitryduev) [reply tox.ini:10]: I think tox supports nested substitutions, let's give it a spin and see :)

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Looks great, thanks @manangoel99! Please see my comments +
- Please fix linting (flake8 complains - see https://app.circleci.com/pipelines/github/wandb/client/12965/workflows/f9b25f8f-dd86-468f-a77f-2d6a276944f2/jobs/198184)
- You need to update the CircleCI config to actually run these new `toxenv`'s here: https://github.com/wandb/client/blob/master/.circleci/config.yml#L936

File: tox.ini (hunk 1 above)
Comment (manangoel99) [reply tox.ini]: Looks like it does!

**Comment by manangoel99** [comment]
@dmitryduev I've made the suggested changes

File: tox.ini [hunk 3]
```diff
@@ -395,7 +395,7 @@ commands =
     cp .coverage coverage.xml cover-results/
     coverage report -m --ignore-errors --skip-covered --omit "wandb/vendor/*"
 
-[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
+[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,imports8,imports9,imports10,imports11,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
```
Comment (dmitryduev) [inline tox.ini:398]:
```suggestion
[testenv:func-s_{base,sklearn,metaflow,tf{115,21,25,26},service,py310,docs,imports{1,2,3,4,5,6,7,8,9,10,11},noml,grpc,kfp}-{py{36,37,38,39,310}}]
```

**Review by dmitryduev — APPROVED** [review]
Thanks @manangoel99, looks great!

File: tox.ini [hunk 4]
```diff
@@ -395,7 +395,7 @@ commands =
     cp .coverage coverage.xml cover-results/
     coverage report -m --ignore-errors --skip-covered --omit "wandb/vendor/*"
 
-[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
+[testenv:func-s_{base,sklearn,metaflow,tf{115,21,25,26},service,py310,docs,imports{1,2,3,4,5,6,7,8,9,10,11},noml,grpc,kfp}-{py{36,37,38,39,310}}]
```
Comment (dmitryduev) [inline tox.ini:398]:
```suggestion
[testenv:func-s_{base,sklearn,metaflow,tf115,tf21,tf25,tf26,service,py310,docs,imports1,imports2,imports3,imports4,imports5,imports6,imports7,imports8,imports9,imports10,imports11,noml,grpc,kfp}-{py36,py37,py38,py39,py310}]
```

**Pull request merged**